.github/workflows/build.yml

```diff
@@ -10,10 +10,28 @@ on:
 
 env:
   CI: true
+  DOCKER_REPO: renovate/renovate
+  YARN_CACHE_KEY: v1
+  BUILDX_VERSION: v0.3.1
+  # https://github.com/chalk/supports-color/issues/106
+  FORCE_COLOR: true
 
 jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+
+    steps:
+      - name: github-cleanup
+        uses: renovatebot/gh-action@v0
+        continue-on-error: true
+        with:
+          command: github-cleanup
+          token: ${{ secrets.GH_TOKEN }}
+          dry-run: ${{github.ref == 'refs/heads/master'}}
```

Comment:
Only dry-run on master, so no cancel. but we get warning if there is a previous running workflow. So we can see  an check if required.